### ISSUE TITLE
Align Filestream Configuration to all Hints templates

### ISSUE THREAD

**Issue by ycombinator** [elastic/elastic-agent#6372]
_Cloned from #5686 as that issue now contains a broader discussion about hints templates._

We need to update the prospector block confguration in hint templates

The proposed Filestream configuration should be https://github.com/elastic/integrations/pull/11212

But for eg in logs template 
```yaml
prospector:
  scanner:
    fingerprint.enabled: true
    symlinks: true
file_identity.fingerprint: ~
```

This is the referencing pipeline responsible to build the packages: https://github.com/elastic/package-storage-infra/blob/main/.buildkite/pipeline.refresh-kubernetes-templates.yml
(Way to trigger it manually [here](https://github.com/elastic/package-storage-infra/pull/169#issue-1303090936)

Where the prospector is defined: https://github.com/ChrsMark/package-storage-infra/blob/54af49b06af4f7e6423574bd0c04206e6ab5e45d/jobs/refreshk8stemplates/job.go#L488-L512

**Comment by swiatekm** [comment]
https://github.com/elastic/package-storage-infra/pull/888 is not enough, I need to actually run the job now and merge the PR in the agent repo.

**Comment by swiatekm** [comment]
Resolved in #6402.
